data/currentGames.json

```diff
@@ -0,0 +1 @@
+[]
```

Comment:
Does this file need to be checked in? If the file doesn't exist, does the bot know to create it?

Reply:
I'm not sure how to do that actually

Reply:
I don't know either. Just seems like an environment specific file.